.circleci/config.yml

```diff
@@ -137,12 +137,24 @@ jobs:
     steps:
       - setup_vm
       - run:
-          name: 'Install Validator Dependencies'
-          command: ./.circleci/install_validator_dependencies.sh
+          name: 'Install Protobuf'
+          command: pip3 install --user protobuf
       - run:
           name: 'Validator Tests'
           command: node build-system/pr-check/validator-tests.js
       - fail_fast
+  'Validator C++ Tests':
+    executor:
+      name: amphtml-medium-executor
+    steps:
+      - setup_vm
+      - run:
+          name: 'Install Validator C++ Dependencies'
+          command: ./.circleci/install_validator_dependencies.sh
+      - run:
+          name: 'Validator C++ Tests'
+          command: node build-system/pr-check/validator-cpp-tests.js
+      - fail_fast
```

Comment:
Separating the JS and CPP builds will start up an extra VM that does nothing most of the time. This will have an impact on overall CI throughput, since we have a limited number of VMs in total for the entire `ampproject` organization.

A better approach would be to go back to a consolidated 'Validator Tests` job, but use an `xlarge` VM.

Reply:
Ok, going back to a consolidated validator test